### PR TITLE
Bug Fixes and GameScene Cleanup

### PR DESCRIPTION
**Pull request by ammarasyad**
- Fixed a bug where if you disable complex effects without restarting the app the combo stays at 0.
- Fixed looping userpanel toast message when not online.
- Fixed score text not displaying 9th and 10th digit. Quick patch, might change method later.
- Cleaned up GameScene.
- Reverted the avatar loading fix as a major overhaul in the Options screen is expected.
- Added a toast message upon startup completion if storage is low which will warn users of possible replay corruption.

and a default clause in a switch so codefactor can stop messing around

### CODE REVIEW

File: src/ru/nsu/ccfit/zuev/osu/MainActivity.java [hunk 1]
```diff
@@ -298,13 +297,39 @@ public void onComplete() {
...
+            availableMemory = (double) stat.getAvailableBytes();
+        } else {
+            long blockSize = stat.getBlockSize();
+            long availableBlocks = stat.getAvailableBlocks();
+            availableMemory = (double) (availableBlocks * blockSize);
+        }
+        if(availableMemory < 0.5*minMem) { //I set 512MiB as a minimum
+            Toast.makeText(this, "Storage space is low. Free Storage: " + df.format(availableMemory / minMem)
+                    +  " GB. This may cause replay/score submission bugs.", Toast.LENGTH_SHORT).show();
+        }
```
Comment (EdrowsLuo) [inline src/ru/nsu/ccfit/zuev/osu/MainActivity.java:330]: do not use hard coded string

**Review by EdrowsLuo — CHANGES_REQUESTED** [review]

File: src/ru/nsu/ccfit/zuev/osu/MainActivity.java (hunk 1 above)
Comment (ammarasyad) [reply src/ru/nsu/ccfit/zuev/osu/MainActivity.java:330]: oh yeah i forgot, lemme do a quick change

File: src/ru/nsu/ccfit/zuev/osu/game/GameScene.java [hunk 2]
```diff
@@ -225,7 +191,7 @@ private void setBackground() {
                 scene.setBackground(new SpriteBackground(bgSprite));
             } else if (pars.length >= 3 && pars[0].equals("2")) {
 
-            } else if (bgset == false && pars.length == 5
+            } else if (!!bgset && pars.length == 5
                     && pars[0].equals("3") && pars[1].equals("100")) {
```
Comment (EdrowsLuo) [inline src/ru/nsu/ccfit/zuev/osu/game/GameScene.java:195]: "!!bgset" ? "!!" should be removed

Comment (ammarasyad) [reply src/ru/nsu/ccfit/zuev/osu/game/GameScene.java:195]: oops, typo

**Comment by ammarasyad** [comment]
The merged GameScene conflicts with my bug fix, I'll fix it.

**Comment by ammarasyad** [comment]
Yknow what I'll make a new PR.

**Pull request closed without merging**